### PR TITLE
Sell golden tenches for molch pearls

### LINKED ISSUE #3999: Aerialfishing lost ability to sell Golden Tenches for 100 pearls.
## 🐛 Bug Report

https://wiki.oldschool.gg/skills/fishing/aerial-fishing

says the following: You can sell Golden Tenches for 100 Molch pearls via +aerialfish sell golden tench

this was lost when the Aerial Fishing was turned into /, please return it.

### PR DESCRIPTION
**Pull request by TastyPumPum**
Adds the ability to sell golden tenches for molch pearls, this was lost when moved to mahoji.
Closes #3999

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

**Review by gc — CHANGES_REQUESTED** [review]
What if someone tries to sell a tbow and a golden tench at the same time? do the same thing we do for mole claws

**Review by gc — CHANGES_REQUESTED** [review]

**Comment by TastyPumPum** [comment]
> 

I believe I have done this how you wanted now, if I haven't some further guidance would be appreciated instead of an emoji. Tested with trying to sell multiple items and seems to mirror the mole claws/skins.

File: src/mahoji/commands/sell.ts [hunk 1]
```diff
@@ -101,6 +101,23 @@ export const sellCommand: OSBMahojiCommand = {
 			return `You exchanged ${moleBank} and received: ${loot}.`;
 		}
 
+		if (bankToSell.has('Golden tench')) {
+			const loot = new Bank();
+			const tenchBank = new Bank();
+			tenchBank.add('Golden tench', bankToSell.amount('Golden tench'));
+			for (let i = 0; i < tenchBank.amount('Golden tench'); i++) {
+				loot.add('Molch pearl', 100);
+			}
```
Comment (gc) [inline src/mahoji/commands/sell.ts:110]: `loot.add(tenchBank.amount() * 100)` ?

**Review by gc — CHANGES_REQUESTED** [review]
The hamstare was because of blatantly untested code (which was fixed)

**Comment by TastyPumPum** [comment]
Sorry I should have marked it as a draft at that point, I was on mobile and done them changes so it would prompt me on pc, I didn't expect you to come back to it so quick without me replying or anything.

**Pull request merged**